IoTEdgeModbus/Modules/AzureIoTEdgeModbus/Slave/ModbusOutContent.cs

```diff
@@ -1,10 +1,16 @@
-﻿namespace AzureIoTEdgeModbus.Slave
+﻿using Newtonsoft.Json;
```

Comment:
Please move all using statements within the namespace curly braces, this is a convention in this code base. Currently in this code we have one "using" statement outside and one inside the namespace curly braces.

Reply:
Fixed